tests/test_libnetplan.py

```diff
@@ -678,3 +678,18 @@ def test_dump_yaml_absent_key(self):
             libnetplan.dump_yaml_subtree('network\tethernets\teth0', f, output)
             output.seek(0)
             self.assertEqual(yaml.safe_load(output), None)
+
+    def test_validation_error_exception(self):
+        parser = libnetplan.Parser()
```

Comment:
Yeah we probably should better document our tests... :sweat_smile: 

In this case it will raise ValidationException because "set-name" requires the use of "match".